app/src/main/java/com/main/mealplanner/MainViewModel.kt

```diff
@@ -10,8 +10,8 @@ class MainViewModel : ViewModel() {
     var recipes: MutableLiveData<ArrayList<RecipeHeader>> = MutableLiveData()
     var recipeService: RecipeService = RecipeService()
 
-    fun fetchRecipe(s: String) {
```

Comment:
Definitely rename this parameter variable to something more descriptive. I renamed it mealName.